bluemira/geometry/parameterisations.py

```diff
@@ -232,6 +234,111 @@ def from_json(cls, file: Union[str, TextIO]) -> GeometryParameterisation:
         var_dict = json.load(file)
         return cls(var_dict)
 
+    def _annotator(self, ax, key: str, xy1: Tuple, xy2: Tuple, xy3: Tuple):
+        """
+        Create annotation arrow with label
+
+        Parameters
+        ----------
+        ax: Axes
+            Matplotlib axis instance
+        key: str
+            label of annotation
+        xy1: Tuple
+            Tuple for first arrow point
+        xy2: Tuple
+            Tuple for second arrow point
+        xy3: Tuple
+            Tuple for arrow label location
+
+        """
+        key = key.replace("_", "")  # TODO proper latex
```

Comment:
Another TODO here. If it's important to do, then we should probably open an issue, or we'll forget about it.